### PR TITLE
chore: configure stack management actions

### PR DESCRIPTION
**Pull request by hspitzley-czi**

### CODE REVIEW

File: .github/workflows/argus-delete-stack-rdev.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  delete-rdev-stack:
+    runs-on: [ARM64, self-hosted, Linux]
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'stack') || github.event.pull_request.state == 'closed' }}
+    permissions:
+        id-token: write
+        contents: read
+
+    steps:
```
Comment (jakeyheath) [inline .github/workflows/argus-delete-stack-rdev.yaml:15]: @hspitzley-czi can you change the BASE_URL to staging so we can register them in staging first?

Comment (hspitzley-czi) [reply .github/workflows/argus-delete-stack-rdev.yaml:15]: current default is staging

Comment (hspitzley-czi) [reply .github/workflows/argus-delete-stack-rdev.yaml:15]: when we deploy the api to prod i'll change the default

Comment (jakeyheath) [reply .github/workflows/argus-delete-stack-rdev.yaml:15]: 👍🏻

File: .github/workflows/argus-delete-stack-rdev.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      - name: Create Stack
```
Comment (hspitzley-czi) [inline .github/workflows/argus-delete-stack-rdev.yaml:16]:
```suggestion
      - name: Delete Stack
```

File: .github/workflows/argus-create-stack-rdev.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  create-rdev-stack:
+    runs-on: [ARM64, self-hosted, Linux]
+    if: contains(github.event.pull_request.labels.*.name, 'stack')
+    permissions:
+        id-token: write
+        contents: read
+
+    steps:
+      - name: Create Stack
+        uses: chanzuckerberg/core-platform/ci/packages/create-stack@ci-v0.6.2
```
Comment (hspitzley-czi) [inline .github/workflows/argus-create-stack-rdev.yaml:17]:
```suggestion
        uses: chanzuckerberg/core-platform/ci/packages/create-stack@ci-v0.7.0
```

File: .github/workflows/argus-delete-stack-rdev.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  delete-rdev-stack:
+    runs-on: [ARM64, self-hosted, Linux]
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'stack') || github.event.pull_request.state == 'closed' }}
+    permissions:
+        id-token: write
+        contents: read
+
+    steps:
+      - name: Delete Stack
+        uses: chanzuckerberg/core-platform/ci/packages/delete-stack@v0.6.2
```
Comment (hspitzley-czi) [inline .github/workflows/argus-delete-stack-rdev.yaml:17]:
```suggestion
        uses: chanzuckerberg/core-platform/ci/packages/delete-stack@v0.7.0
```

**Pull request merged**